### ISSUE TITLE
cannot re-implement the result on the paper.

### ISSUE THREAD

**Issue by archfelix** [facebookresearch/silk#73]
I have re-implemented a Torch-Only version according to the paper:
https://github.com/archfelix/SiLK_torch_simple

But I cannot get what the problem it has. I have tried:
1. Use ReLu and LeakyReLu
2. Use torch.nn.functional.cosine_similarity and torch.matmul
3. tune the Tau parameter
4. Check whether Pij is right
5. Added a weight=200 before the Positive sample, to balance:
        return torch.sum(y*200*torch.log(F.sigmoid(q[c])) + (1-y) * torch.log(F.sigmoid(-q[c]))) / -N

I really cannot find where the problem is, please help!

**Comment by gleize** [comment]
Solved in https://github.com/archfelix/SiLK-Lite/issues/1.
Closing now.
